File: javascript-exercicios/ex048.js
let n_id = 0
let s = 0
for(let c = 1; c < 501; c+=2){
    if(c%3==0){
        s+=c
        n_id+=1
    }
}
console.log(`a soma de todos os ${n_id} valores identificados entre 1 e 500 é: ${s}`)

File: curso-python/ex048.py
#Exercício Python 48: Faça um programa que calcule a soma entre todos os números ímpares que são múltiplos de três e que se encontram no intervalo de 1 até 500.

numeros_identificados = 0
soma = 0
for c in range(1, 501, 2):
    if c % 3 == 0:
        soma += c
        numeros_identificados += 1


print(f"a soma de todos os {numeros_identificados} valores solicitados entre 1 e 500 é: {soma}")